### ISSUE TITLE
Train large model (real)

### ISSUE THREAD

**Issue by nadahlberg** [freelawproject/classifier-experiments#95]
Finetune modernbert large using the best learning rate from the experiments. Use a larger dataset and tune until model converges.
